articles/fleet-4.19.0.md

```diff
@@ -0,0 +1,83 @@
+- See remaining disk space for all hosts
+- Aggregate Windows and mobile device management (MDM) data
+
+## Just-in-time (JIT) user creation
+**Available in Fleet Premium**
+
+Just-in-time user provisioning makes it easier to spread the power of Fleet and osquery throughout your organization. 
+
+Fleet will now automatically create a user account when a new user attempts to log in to Fleet via single sign-on (SSO). New user accounts are given the Observer role. For more information on roles and their permissions in Fleet, check out the documentation [here on fleetdm.com/docs](https://fleetdm.com/docs/using-fleet/permissions).
+
+## See remaining disk space for all hosts
+**Available in Fleet Free & Fleet Premium**
+
+![See remaining disk space for all hosts](../website/assets/images/articles/fleet-4.19.0-1.gif)
+
+Fleet 4.19.0 adds the ability to quickly see which hosts may not have enough remaining disk space to install operating system updates.
+
+To see this in action in Fleet, head to the **Home > macOS** page and hover over a row in the **Operating systems** table. Select **View all hosts** to navigate to a list of hosts a specific operating system version installed. Finally, select the **Disk space available** column to sort hosts by least remaining disk space to most.
+
+## Aggregate Windows and mobile device management (MDM) data
+**Available in Fleet Free & Fleet Premium**
+
+![Aggregate Windows data](../website/assets/images/articles/fleet-4.19.0-3-1200x675@2x.jpg)
+
+Fleet 4.19.0 adds the ability to see a list of all Windows operating system (OS) versions installed across all your hosts. 
+
+Also, Windows operating system versions are improved by including the display version like "21H" or "21H2." This allows you to quickly reference [endoflife.date/windows](https://endoflife.date/windows) to see which OS's Microsoft no longer supports. 
+
+![Aggregate mobile device management (MDM) data](../website/assets/images/articles/fleet-4.19.0-2.gif) 
+
+Fleet 4.19.0 also adds the improved Windows operating system versions on the **Hosts** page and **Host details** page in the Fleet UI as well as the `GET /hosts` and `GET /hosts/{id}` API routes.
+
+This release also adds the ability to see a list of all MDM solutions your hosts are enrolled to. Head to the **Home > macOS** page in Fleet to check out how your migration from one MDM solution to another is going.
+
+## More new features, improvements, and bug fixes
+
+In 4.19.0, we also:
+
+* Improved performance for aggregating software inventory. Aggregate software inventory is displayed on the **Software page** in the Fleet UI.
+
+* Added the ability to see the vendor for Windows programs in software inventory. Vendor data is available in the [`GET /software` API route](https://fleetdm.com/docs/using-fleet/rest-api#software).
+
+* Added a message in `fleetctl` to that notifies users to run `fleet prepare` instead of `fleetctl prepare` when running database migrations for Fleet.
```

Comment:
```suggestion
* Added a message in `fleetctl` that notifies users to run `fleet prepare` instead of `fleetctl prepare` when running database migrations for Fleet.
```